docs/backend/guides/tget/index.md

```diff
@@ -0,0 +1,33 @@
+---
+id: index
+sidebar_position: 1
+slug: /backend/guides/tget
+---
+
+# TGET
+
+TGET stands for Transportation Global Edit Table. Each service has their own set of valid accounting codes, also known as TAC (transportation accounting codes).
+
+An example of this would be where the TOO (Transportation Ordering Officer) enters a TAC before approving a shipment. A TAC determines who pays for the shipping service (from which account). This TAC needs to be verified as a valid TAC. The MilMove application uses these to make sure that a customers order is valid. However, it should be noted that these codes do change and are updated.
+
+## Context
+
+The current implmentation is based from a sample file that had been parsed to retrieve all the unique TAC values. A new table in the database was made called `transportation_accounting_codes` and a secure migration was done to put all those unique values into the database.
+
+In the MilMove app, when a user enters a value in to the TAC field, a call gets made to our backend to check if the input shows up in the database.
```

Comment:
**suggestion (non-blocking)**: I'm not sure if this is the best place, or if we even want this in this page, but something to note is that we don't require a TAC to be valid in order to proceed. We let the user know if it isn't valid, but they're allowed to proceed.

Reply:
That's a good thing to highlight here. I'll add it in.